packages/@ember/-internals/glimmer/lib/helpers/if-unless.ts

```diff
@@ -13,9 +13,9 @@ function ifHelper({ positional }: CapturedArguments) {
     positional.length === 3 || positional.length === 2
   );
 
-  let condition = positional.at(0);
-  let truthyValue = positional.at(1);
-  let falsyValue = positional.at(2);
+  let condition = positional[0];
+  let truthyValue = positional[1];
+  let falsyValue = positional[2];
```

Comment:
```suggestion
  let [ condition, truthyValue, falsyValue ] = positional;
```

Now that it is just an array, we can destructure...